scripts/disable-resource-locking.sh

```diff
@@ -1,24 +1,37 @@
 #!/bin/bash
 set -e
 
-JSONPATH="[?(contains(type, 'Storage') || contains(type, 'KeyVault') || contains(type, 'SQL') || contains(type, 'Insights')) ].[resourceGroup]"
+RESOURCETYPES_LIST=$1
```

Comment:
```suggestion
RESOURCE_GROUPS_LIST=$1
```

Reply:
As discussed passing in the RGs would be much cleaner as we would know the specific Resource group which needs unlocking. Even otherwise the enable lock cronjob will be scheduled to put the missing locks back in place.